sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -766,7 +766,7 @@ ProgramManager::build(ProgramPtr Program, const ContextImplPtr Context,
   // is built during piProgramCreate.
   // TODO: remove this check as soon as piProgramCompile/piProgramLink will be
   // implemented in L0 plugin.
```

Comment:
L0 -> Level-Zero, and please grep for other instances of "L0"

Reply:
Done.